ds_judgements_public_ui/templates/includes/cards_homepage.html

```diff
@@ -1,12 +1,12 @@
 {% load static %}
 <ul class="cards-homepage">
   <li>
-    {% include "includes/card_homepage.html" with title="Recently published" body="Browse legal decisions most recently added to the Find Case Law site." link_route="search" image_src="/images/recently_added.jpg" %}
+    {% include "includes/card_homepage.html" with title="Browse by court or tribunal" body="Browse judgments and tribunals from the United Kingdoms's 18 courts and 7 tribunals." link_route="courts_and_tribunals" image_src="/images/court-crest.png" %}
```

Comment:
Sceptical of this text -- is that how many courts/tribunals there are or how many we have? Because I'm pretty sure there's a mismatch somewhere.

Reply:
@dragon-dxw this is same text used previously on the cards homepage not changed it maybe best to ask Rose.

Reply:
@dragon-dxw text now updated with content done by Ashley.